cla-backend-go/swagger/cla.v2.yaml

```diff
@@ -4276,6 +4296,24 @@ parameters:
     pattern: '^[a-fA-F0-9]{8}-?[a-fA-F0-9]{4}-?4[a-fA-F0-9]{3}-?[89ab][a-fA-F0-9]{3}-?[a-fA-F0-9]{12}$' # uuidv4
     minLength: 5
     maxLength: 255
+  path-gitlabOrganizationID:
+    name: organizationID
+    description: gitlab organization ID
```

Comment:
Let's make this description consistent with the others, make it say 'GitLab organization ID'.  I am not good with this either, but together can clean up the documentation generated from the swagger build process.